src/javaRosa.js

```diff
@@ -218,7 +224,6 @@ define([
         util.eventuality(this);
         
         this.languages = [];
-        this.items = [];
```

Comment:
The ItextModel no longer keeps a list of items. Itext items are now always accessed via mugs.